xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/documentTags.vm

```diff
@@ -143,7 +143,16 @@
         #displayTag($tag)
     #end
     #if($hasedit && $xwiki.tag)
-        <div class="tag-tool tag-add">#if("$!{request.showTagAddForm}" == '')<a href="$doc.getURL('view', "showTagAddForm=true&amp;$!{escapetool.url(${request.queryString})}#${tagsId}")" title="$services.localization.render('core.tags.add.tooltip')" rel="nofollow">[+]</a>#else #displayAddForm()#end</div>
+        <div class="tag-tool tag-add">##
+            #if("$!{request.showTagAddForm}" == '')
+                <a href="$doc.getURL('view', 
+                    "showTagAddForm=true&amp;$!{escapetool.url(${request.queryString})}#${tagsId}")" 
+                    title="$services.localization.render('core.tags.add.tooltip')" 
+                    rel="nofollow">[+]</a>##
+            #else
+                #displayAddForm()
+            #end
+        </div>##
```

Comment:
That last one is not mandatory, it wasn't here before the code style improvement.